src/main/java/com/epam/controller/ReservationController.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.controller;
+
+import com.epam.model.Reservation;
+import com.epam.service.ReservationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.security.Principal;
+import java.util.List;
+
+@Controller
+public class ReservationController {
+    @Autowired
+    ReservationService reservationService;
```

Comment:
this variable can be private final and autowired using constructor, because it is easy t understand and write tests

Reply:
done